net/dnscache/dnscache.go

```diff
@@ -108,6 +112,19 @@ var debug = envknob.Bool("TS_DEBUG_DNS_CACHE")
 // If err is nil, ip will be non-nil. The v6 address may be nil even
 // with a nil error.
 func (r *Resolver) LookupIP(ctx context.Context, host string) (ip, v6 net.IP, allIPs []net.IPAddr, err error) {
```

Comment:
this signature continues to be stupid & old (predating netaddr.IP).

it's on my list to clean up.